packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions-tasks/test/emr/integ.emr-create-cluster-with-spot-instance-fleet.ts

```diff
@@ -0,0 +1,43 @@
+import * as sfn from 'aws-cdk-lib/aws-stepfunctions';
+import { App, Stack } from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { EmrCreateCluster } from 'aws-cdk-lib/aws-stepfunctions-tasks';
+
+const app = new App();
+
+const stack = new Stack(app, 'aws-cdk-emr-create-cluster-with-spot-instance-fleet');
+
+const step = new EmrCreateCluster(stack, 'EmrCreateCluster', {
+  releaseLabel: 'emr-5.36.1',
+  instances: {
+    instanceFleets: [{
+      instanceFleetType: EmrCreateCluster.InstanceRoleType.MASTER,
+      instanceTypeConfigs: [{
+        instanceType: 'm5.xlarge',
+        weightedCapacity: 1,
+      }],
+      launchSpecifications: {
+        spotSpecification: {
+          allocationStrategy: EmrCreateCluster.SpotAllocationStrategy.CAPACITY_OPTIMIZED,
+          timeoutAction: EmrCreateCluster.SpotTimeoutAction.TERMINATE_CLUSTER,
+          timeoutDurationMinutes: 60,
+        },
```

Comment:
`blockDurationMinutes` is not specified, because it is no longer available.

```
Spot Instances with a defined duration (also known as Spot blocks) are no longer available to new customers from July 1, 2021. For customers who have previously used the feature, we will continue to support Spot Instances with a defined duration until December 31, 2022.
```

https://docs.aws.amazon.com/emr/latest/APIReference/API_SpotProvisioningSpecification.html